src/app/modules/tiles/tiles.component.html

```diff
@@ -5,17 +5,17 @@
        [class.wow]="fadeIn(i)"
        [class.fadeIn]="fadeIn(i)"
        [attr.data-wow-delay]="fadeIn(i)+'s'">
-    <div class="team-image" relative
+    <div class="team-photo" relative
```

Comment:
Is there a reason to change the class name? If so, did you make sure you have updated all the places where this class is used?

Reply:
This was a mistake. Corrected it.